docs/how-tos/train-deploy-ml.md

```diff
@@ -299,11 +299,25 @@ If no data appears after the sync interval, check the **LOGS** tab for errors.
 Your camera now saves images at the configured time interval.
 When training machine learning models, it is important to supply a variety of images.
 The dataset you create should represent the possible range of visual input.
-This may include capturing images of different angles, different configurations of objects and different lighting conditions
+This may include capturing images of different angles, different configurations of objects and different lighting conditions.
 The more varied the provided dataset, the more accurate the resulting model becomes.
 
 Capture at least 10 images of anything you want your machine to recognize.
 
+{{< expand "For more tips and tricks on improving model accuracy, click here." >}}
+
+- **More data means better models:** Incorporate more data from varying environments, such as lighting conditions or distances, to improve your model’s overall performance.
+- **Include counterexamples:** Include images with and without the object you’re looking to classify.
+  This helps the model distinguish the target object from the background and reduces the chances of false positives by teaching it what the object is not.
+- **Avoid class imbalance:** Don’t train excessively on one specific type or class, make sure each category has a roughly equal number of images.
+  For instance, if you're training a dog detector, include images of various dog breeds to avoid bias towards one breed. An imbalanced dataset can lead the model to favor one class over others, reducing its overall accuracy.
```

Comment:
```suggestion
  For instance, if you're training a dog detector, include images of various dog breeds to avoid bias towards one breed.
  An imbalanced dataset can lead the model to favor one class over others, reducing its overall accuracy.
```
[nit]